src/main/java/com/github/krzysiekjodlowski/ox/Player.java

```diff
@@ -0,0 +1,61 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+import com.github.krzysiekjodlowski.ox.ui.UI;
+
+/**
+ * Base for any player. Each one descendant
+ * will have to implement its own version
+ * of making moves and adjust toString.
+ *
+ * @author Krzysztof Jodlowski
+ */
+abstract class Player {
```

Comment:
Maybe some some Score field (you don't have it so - maybe some Score class 😃 ).

Reply:
Yes! That is the plan for "the next" step of this project;) Score for the Player, Round for the Game, Result keeping track of the Scores, Settings downloading default settings and Languages...